packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/aws-custom-resource/integ.aws-custom-resource-vpc.ts

```diff
@@ -30,9 +30,18 @@ new AwsCustomResource(stack, 'DescribeVpcAttribute', {
   vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_EGRESS },
 });
 
-new IntegTest(app, 'CustomResourceVpc', {
+const integTest = new IntegTest(app, 'CustomResourceVpc', {
   testCases: [stack],
   diffAssets: true,
 });
 
-app.synth();
+// Assertion for the custom resource
+integTest.assertions.awsApiCall('EC2', 'describeVpcAttribute', {
```

Comment:
I don't think this assertion makes sense given the integ test. An assertion should be something we're using to verify that an action was taken successfully during the integ test. In this case, how can we assert that the previous `describeVpcAttribute` worked correctly? Making another separate `describeVpcAttribute` call is decoupled from the one that took place in the integ test. In this case, I don't think we have anything to assert against.

Reply:
Okay, so is there any other way to test it out? or since `describeVpcAttribute` call is decoupled from the previous one should I just remove it and leave it as is?

Reply:
Just remove it and leave it as is.